### PR TITLE
New FAQ page

### LINKED ISSUE #2016: Create seperate FAQ page
The drop down menu on the site is getting too long.

### PR DESCRIPTION
**Pull request by nlharris**
replaces PR #2015 (which added the new dashboard section to the FAQ menu).
fixes #2016

### CODE REVIEW

**Comment by cthoyt** [comment]
How would a user know to navigate to this page? Shouldn't this PR also remove the entire drop-down list from the navbar?

**Comment by nlharris** [comment]
i was going to do that part after this page is merged so that i can check that it looks ok.

File: faq/index.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+---
+layout: faq
+title: FAQs
+---
+
+## OBO Foundry Frequently Asked Questions (FAQs)
```
Comment (cthoyt) [inline faq/index.md:6]: this is taken care of by the `title` in the front matter and is therefore redundant

File: faq/index.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+### Policy
+- <a href="/principles/fp-000-summary.html">What are the OBO Foundry Principles?</a>
```
Comment (cthoyt) [inline faq/index.md:9]: would prefer using markdown style links where possible

Comment (nlharris) [reply faq/index.md:9]: totally agree but i converted the menu to this page using a few regexps. to rewrite it in md would require either more extensive scripting or tedious editing. or i guess i could try an html-to-md converter, if you feel strongly about it.

File: faq/index.md (hunk 1 above)
Comment (nlharris) [reply faq/index.md]: thanks, fixed

File: faq/index.md (hunk 2 above)
Comment (cthoyt) [reply faq/index.md:7]: it's not a big deal but markdown syntax is much more straightfowards and less error-prone.

Comment (nlharris) [reply faq/index.md:7]: if i had created the page from scratch i would have used md. it didn't seem worth it to me to spend a lot of time to convert the html to md -- but if you would like to do that, please feel free! 🙂

**Review by matentzn — APPROVED** [review]
This is great, thank you @nlharris and @cthoyt!

**Pull request merged**